devito/passes/clusters/implicit.py

```diff
@@ -106,6 +107,13 @@ def callback(self, clusters, prefix):
             # The "implicit expressions" created for the MultiSubDomain
             exprs, dims, sub_iterators = make_implicit_exprs(d.msd, c)
 
+            # Make sure the "implicit expressions" aren't scheduled in
+            # an inner loop. E.g schedule both for `t, xi, yi` and `t, d, xi, yi`
```

Comment:
what do you mean by "both" here? 
I think the example could be a bit more clear here
OK in another PR

Reply:
For clusters with itspace `t, xi, yi` and `t, d, xi, yi` then this statement will be reached twice during the `self.process` once for `t` as a prefix and once for `t, d`